### PR TITLE
Add documentation for CustomAttrbute

### PR DESCRIPTION
**Pull request by MSDN-WhiteKnight**
Add remarks and example for System.Reflection.Metadata.CustomAttrbute

### CODE REVIEW

**Review by BillWagner — COMMENTED** [review]
Commenting to give @gewarren final review / merge decision.

I've read through the code and the text, and I think it's good to merge.

File: xml/System.Reflection.Metadata/CustomAttribute.xml [hunk 1]
```diff
@@ -31,8 +31,20 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides information about the custom attribute.</summary>
```
Comment (gewarren) [inline xml/System.Reflection.Metadata/CustomAttribute.xml:34]:
```suggestion
    <summary>Provides information about a custom attribute.</summary>
```

File: xml/System.Reflection.Metadata/CustomAttribute.xml [hunk 2]
```diff
@@ -31,8 +31,20 @@
...
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+A custom attribute is an annotation that associates additional information with a metadata element, such as assembly, type, or method. You can use the <xref:System.Reflection.Metadata.MetadataReader.GetCustomAttribute(System.Reflection.Metadata.CustomAttributeHandle)> method to get a custom attribute instance. For more information about attributes in .NET, see [Extend metadata using attributes](/dotnet/standard/attributes/).
```
Comment (gewarren) [inline xml/System.Reflection.Metadata/CustomAttribute.xml:39]:
```suggestion
A custom attribute is an annotation that associates additional information with a metadata element, such as an assembly, type, or method. You can use the <xref:System.Reflection.Metadata.MetadataReader.GetCustomAttribute(System.Reflection.Metadata.CustomAttributeHandle)> method to get a custom attribute instance. For more information about attributes in .NET, see [Extend metadata using attributes](/dotnet/standard/attributes/).
```

File: snippets/csharp/System.Reflection.Metadata/CustomAttribute/Overview/CustomAttributeSnippets.cs
[the remark this answers is not in the document]
Comment (buyaa-n) [reply snippets/csharp/System.Reflection.Metadata/CustomAttribute/Overview/CustomAttributeSnippets.cs:1]: Not sure what you meant ...

**Review by buyaa-n — APPROVED** [review]
LGTM, thanks!

File: snippets/csharp/System.Reflection.Metadata/CustomAttribute/Overview/CustomAttributeSnippets.cs
[the remark this answers is not in the document]
Comment (buyaa-n) [reply snippets/csharp/System.Reflection.Metadata/CustomAttribute/Overview/CustomAttributeSnippets.cs:1]: Not sure where / how exactly to add, or maybe add a link to [Create custom attributes](https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/reflection-and-attributes/creating-custom-attributes) or [Write custom attributes](https://learn.microsoft.com/en-us/dotnet/standard/attributes/writing-custom-attributes) in the readme?

[the remark this answers is not in the document]
Comment (MSDN-WhiteKnight) [reply snippets/csharp/System.Reflection.Metadata/CustomAttribute/Overview/CustomAttributeSnippets.cs:1]: I think what @gewarren means is to define attribute class in sample code. Currently this sample, if you run project, will print a bunch of compiler-generated attributes. I'll address this soon.

**Review by gewarren — APPROVED** [review]

**Pull request merged**